jsonschema-core.md

```diff
@@ -862,7 +862,9 @@ The meta-schema serves two purposes:
 Declaring the vocabularies in use: The `$vocabulary` keyword, when it appears in
 a meta-schema, declares which vocabularies are available to be used in schemas
 that refer to that meta-schema. Vocabularies define keyword semantics, as well
-as their general syntax.
+as their general syntax. By combining various vocabularies, distinct
+sets of keywords can be made available for use in a schema. This collection of
+vocabularies defines a dialect.
 
 Describing valid schema syntax: A schema MUST successfully validate against its
 meta-schema, which constrains the syntax of the available keywords. The syntax
```

Comment:
I noticed that a schema must validate against it's meta-schema in two places. The other place is in the `$schema` section on line 920. I don't have an opinion on which place to say this, but it doesn't need to be both places.

This isn't technically related to this change, but it might be a good time to clean that up since we're making changes in this area anyway. I'm also ok with dealing with this separately.

Reply:
I think I'd drop it from 920 since this is a more general "must validate against the meta-schema regardless of how you get the meta-schema" whereas 920 is specifically targeted to `$schema`.